src/app/home/home.component.html

```diff
@@ -13,7 +13,7 @@
     (click)="navigate('chat')"
     (tap)="navigate('chat')"
   >
-    CHAT
+    CHAT {{ !ownSession && newMessages > 0 && currentTab !== 'chat' ? '(' + newMessages + ')' : '' }}
```

Comment:
`span` is an inline element. That should work. If not, add a class and set this class to `display: inline`.